base/Plain_netCDF_Time.F90

```diff
@@ -287,12 +287,12 @@ subroutine get_att_real_netcdf(filename, varname, att_name, att_value, group_nam
     character(len=*), intent(in) :: varname
     character(len=*), intent(in) :: att_name
     real(REAL64), intent(out) :: att_value
-    character(len=*), optional, intent(out) :: group_name    
+    character(len=*), optional, intent(out) :: group_name
```

Comment:
I've not looked closely, but generally an allocatable string is preferable when `intent(out)` (or result of function.  But don't change if it requires changing lots of other declarations.

Reply:
I will think about the allocatable character in the coding later on. Here there are many hard coded string with fixed length, requiring trim. I will do a clean up in another PR.